### PR TITLE
stm32l4-multi: add mask for reading bytes of size 7 and 6  from tty

### PR DESCRIPTION
**Pull request by BartoroDev**
JIRA: DEND-39

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Review by anglov — APPROVED** [review]
LGTM

**Review by agkaminski — APPROVED** [review]

File: multi/stm32l4-multi/tty.c [hunk 1]
```diff
@@ -309,12 +310,22 @@ static void tty_irqthread(void *arg)
 		}
 		mutexUnlock(ctx->irqlock);
```
Comment (gorbanchovie) [inline multi/stm32l4-multi/tty.c:311]: On imx driver I got such comment:

"imho this check should be performed under mutex"

Maybe it applies also here?

File: multi/stm32l4-multi/tty.c [hunk 2]
```diff
@@ -309,12 +310,22 @@ static void tty_irqthread(void *arg)
...
 
+		if ((ctx->ttyCommon.term.c_cflag & CSIZE) == CS7) {
+			mask = 0x7f;
+		}
+		else if ((ctx->ttyCommon.term.c_cflag & CSIZE) == CS6) {
```
Comment (gorbanchovie) [inline multi/stm32l4-multi/tty.c:316]: You could update commit message with CS6

File: multi/stm32l4-multi/tty.c (hunk 1 above)
Comment (agkaminski) [reply multi/stm32l4-multi/tty.c:311]: In this case we need to synchronize with `tty_setCflag()` - but it is not protected by any lock. So you're right, but this is non-trivial change

Comment (BartoroDev) [reply multi/stm32l4-multi/tty.c:311]: so I will merge it as is, synchronization should be done as an independent change.

**Pull request merged**